components/style/gecko/generated/structs.rs

```diff
@@ -26590,8 +26590,8 @@ pub mod root {
         pub static mut nsCSSProps_kAutoCompletionAlignItems: [root::nsCSSProps_KTableEntry; 0usize];
     }
     extern "C" {
-        #[link_name = "\u{1}_ZN10nsCSSProps34kAutoCompletionAlignJustifyContentE"]
-        pub static mut nsCSSProps_kAutoCompletionAlignJustifyContent:
+        #[link_name = "\u{1}_ZN10nsCSSProps34kAutoCompletionContentDistributionE"]
+        pub static mut nsCSSProps_kAutoCompletionContentDistribution:
```

Comment:
Is this change intended? Please ELI5.

Reply:
Yeah, it is not, but it's harmless anyway.